client/src/components/Atoms/ListAndDetail.tsx

```diff
@@ -49,6 +51,7 @@ const ListItems = styled.ul`
   margin: 0;
   padding: 0;
   padding-bottom: 40px;
```

Comment:
@arsalansufi do you remember what this padding is for? I considered removing it as well

Reply:
Yes good q! I added this because I thought the last list item's bottom border looked a bit off when it touched the bottom section of the wizard:

<img width="546" alt="bottom-border" src="https://user-images.githubusercontent.com/12616928/203372087-ae1ed2a2-129a-47e8-b06f-21125759e849.png">

I think the same idea holds with the border added to the bottom section of the wizard - we'd now have a double border:

<img width="540" alt="double-border" src="https://user-images.githubusercontent.com/12616928/203373137-5904287b-b8e7-4bd2-8a12-0a8779401762.png">

I couldn't just remove the border on the last list item because we need that border when the list isn't tall enough to reach the end of the wizard. A negative 1px margin may do the trick, but the bottom padding also felt like an okay solution to me.

Reply:
Ok that was my guess! I'll leave the padding